CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Change log
 tomland uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
+0.3.1
```

Comment:
Could you please move this to the end of version `0.3` in changelog? Version `0.3` is not released on Hackage yet. So it makes sense to put everything new there.

Also, you should add issue #20 to that changelog.